### PR TITLE
Solved error on html escaped password.

### PR DESCRIPTION
**Pull request by MMaiero**
Code refactoring in GwtNetworkServiceImpl getWifiConfig.
Small refactoring in GwtWifiConfig.

Signed-off-by: MMaiero <matteo.maiero@eurotech.com>

### CODE REVIEW

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/Network/TabWirelessUi.java [hunk 1]
```diff
@@ -1234,40 +1234,40 @@ public void onSuccess(GwtXSRFToken token) {
...
+                                if (name != null && name.equals("devLastWifiChannel")) {
+                                    int topChannel = Integer.parseInt(pair.getValue());
+                                    // Remove channels 12 and 13
+                                    if (topChannel < MAX_WIFI_CHANNEL) {
+                                        try {
+                                            TabWirelessUi.this.channelDataProvider.getList()
+                                                    .remove(MAX_WIFI_CHANNEL - 1);
+                                            TabWirelessUi.this.channelDataProvider.getList()
+                                                    .remove(MAX_WIFI_CHANNEL - 2);
+                                        } catch (UnsupportedOperationException e) {
```
Comment (amitjoy) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/Network/TabWirelessUi.java]: Use multi-catch

**Comment by MMaiero** [comment]
Closes #1369

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtNetworkServiceImpl.java [hunk 2]
```diff
@@ -1367,20 +1368,20 @@ public void updateDeviceFirewallPortForwards(GwtXSRFToken xsrfToken, List<GwtFir
...
             throw new GwtKuraException(GwtKuraErrorCode.INTERNAL_ERROR, e);
         } catch (NumberFormatException e) {
-            s_logger.warn("Exception while updating firewall port forwards", e);
+            logger.warn("Exception while updating firewall port forwards", e);
             throw new GwtKuraException(GwtKuraErrorCode.INTERNAL_ERROR, e);
         } catch (UnknownHostException e) {
-            s_logger.warn("Exception while updating firewall port forwards", e);
+            logger.warn("Exception while updating firewall port forwards", e);
             throw new GwtKuraException(GwtKuraErrorCode.INTERNAL_ERROR, e);
         }
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtNetworkServiceImpl.java]: Use multi-catch also here.

**Review by pierantoniomerlino — APPROVED** [review]
Please fix the multi-catch sections.

**Review by pierantoniomerlino — APPROVED** [review]

**Comment by MMaiero** [comment]
@pierantoniomerlino Done.

**Pull request merged**